test/blackbox/common/BlackboxTestsTransportTCP.cpp

```diff
@@ -732,6 +733,226 @@ TEST_P(TransportTCP, TCPv6_autofill_port)
     EXPECT_TRUE(IPLocator::getPhysicalPort(p2_locators.begin()[0]) == port);
 }
 
+// Test TCPv4 transport on LARGE_DATA topology
+TEST_P(TransportTCP, TCPv4_large_data_topology)
```

Comment:
This is already checking both v4 and v6 (it is a TEST_P)

```suggestion
// Test TCP transport on LARGE_DATA topology
TEST_P(TransportTCP, large_data_topology)
```